external-dependency.md

```diff
@@ -113,18 +111,18 @@
 |k8s.io/utils | Apache License 2.0  |https://github.com/kubernetes/utils
 |paypal/gatt |BSD-3-Clause |https://github.com/paypal/gatt
 |go-ansiterm |MIT |https://github.com/Azure/go-ansiterm
-|win_pdh | |github.com/JeffAshton/win_pdh
-|go-winio |MIT |github.com/Microsoft/go-winio
-|chacha20 |MIT |github.com/aead/chacha20
-|mint |MIT |github.com/bifurcation/mint
-|genny |MIT |github.com/cheekybits/genny
+|win_pdh | |https://github.com/JeffAshton/win_pdh
```

Comment:
extra  |  ?

Reply:
between two `| |` lack the license of win_pdh, the license file in win_pdh repo also don't write which  license it is... could you please help have a double check?:)

Reply:
I've checked and I didn't find the name of the license. I think they have created an own version of the bsd clause 4 license.

Reply:
It's a BSD 3-clause lincense, xref: https://opensource.org/licenses/BSD-3-Clause
checkout the following diff:
```diff
- Copyright <YEAR> <COPYRIGHT HOLDER>
+ Copyright (c) 2010 The win_pdh Authors. All rights reserved.
  
  Redistribution and use in source and binary forms, with or without 
  modification, are permitted provided that the following conditions
  are met:
  1. Redistributions of source code must retain the above copyright
    notice, this list of conditions and the following disclaimer. 
  2. Redistributions in binary form must reproduce the above copyright
    notice, this list of conditions and the following disclaimer in the
    documentation and/or other materials provided with the distribution. 
- 3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products 
+ 3. The names of the authors may not be used to endorse or promote products
    derived from this software without specific prior written permission.
  
- THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
+ THIS SOFTWARE IS PROVIDED BY THE AUTHORS ``AS IS'' AND ANY EXPRESS OR
  IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
  OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
- IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY DIRECT, INDIRECT,
  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
  NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
  THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

```